src/components/CloseButton/CloseButton.driver.ts

```diff
@@ -0,0 +1,36 @@
+import { StylableDOMUtil } from 'stylable/test-utils';
+import { buttonDriverFactory as coreButtonDriverFactory } from 'wix-ui-core/dist/src/components/Button/Button.driver';
+import { DriverFactory, BaseDriver, ComponentFactory } from 'wix-ui-test-utils/driver-factory';
+import style from './CloseButton.st.css';
+import { Skin, Size } from './constants';
+
+// TODO: move this to core
+export interface CoreButtonDriver extends BaseDriver {
+  exists: () => boolean;
+  click: () => any;
```

Comment:
exists is on the BaseDriver, so it is redundant

Reply:
fixed in the Core PR